rkt/image_rm.go

```diff
@@ -95,28 +95,28 @@ func rmImages(s *store.Store, images []string) error {
 		if err := s.RemoveACI(key); err != nil {
 			if serr, ok := err.(*store.StoreRemovalError); ok {
 				staleErrors++
-				stderr("rkt: some files cannot be removed for image %q (%q): %v", key, name, serr)
+				stderr.PrintE(fmt.Sprintf("some files cannot be removed for image %q (%q)", key, name), serr)
 			} else {
-				stderr("rkt: error removing aci for image %q (%q): %v", key, name, err)
+				stderr.PrintE(fmt.Sprintf("error removing aci for image %q (%q)", key, name), err)
 				continue
 			}
 		}
-		stdout("rkt: successfully removed aci for image: %q (%q)", key, name)
+		stdout.Printf("successfully removed aci for image ID: %q", key)
```

Comment:
I would not add the "ID" here - the `image rm` can remove images by name too now.